modules/ximgproc/doc/ximgproc.bib

```diff
@@ -77,3 +77,21 @@ @inproceedings{Farbman2008
   year={2008},
   organization={ACM}
 }
+
+@incollection{zhang2014rolling,
```

Comment:
Reference to this paper should be added to the corresponding filter documentation.